extensions/azure/contract-negotiation-store-cosmos/src/test/java/org/eclipse/dataspaceconnector/contract/negotiation/store/CosmosContractNegotiationStoreTest.java

```diff
@@ -12,23 +12,21 @@
 import java.util.stream.Stream;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.easymock.EasyMock.anyObject;
-import static org.easymock.EasyMock.anyString;
-import static org.easymock.EasyMock.expect;
-import static org.easymock.EasyMock.expectLastCall;
-import static org.easymock.EasyMock.replay;
-import static org.easymock.EasyMock.strictMock;
-import static org.easymock.EasyMock.verify;
 import static org.eclipse.dataspaceconnector.contract.negotiation.store.TestFunctions.generateDocument;
 import static org.eclipse.dataspaceconnector.contract.negotiation.store.TestFunctions.generateNegotiation;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 class CosmosContractNegotiationStoreTest {
     private CosmosContractNegotiationStore store;
     private CosmosDbApi cosmosDbApiMock;
 
     @BeforeEach
     void setup() {
-        cosmosDbApiMock = strictMock(CosmosDbApi.class);
+        cosmosDbApiMock = mock(CosmosDbApi.class);
```

Comment:
since this is was a `strictMock` before, `verifyNoMoreInteractions` would actually make sense here.